lib/active_hash/base.rb

```diff
@@ -1,6 +1,13 @@
 module ActiveHash
 
   class RecordNotFound < StandardError
+    attr_reader :model, :id
+
+    def initialize(message = nil, model = nil, id = nil)
+      @model = model
+      @id = id
+      super(message)
+    end
```

Comment:
is there a reason to not copy the complete `ActiveRecord::RecordNotFound` interface?

```ruby
  class RecordNotFound < ActiveRecordError
    attr_reader :model, :primary_key, :id

    def initialize(message = nil, model = nil, primary_key = nil, id = nil)
      @primary_key = primary_key
      @model = model
      @id = id

      super(message)
    end
  end
```

 seems to make sense to  mimic rails completely.